core/retries/src/test/java/software/amazon/awssdk/retries/internal/RetryStrategyCommonTest.java

```diff
@@ -107,32 +107,32 @@ static Stream<TestCase> buildCases(Function<String, TestCase> defaultTestCaseSup
                               .configure(b -> b.maxAttempts(3))
                               .configure(b -> b.retryOnException(IllegalArgumentException.class))
                               .givenExceptions(IAE, IAE, IAE)
-                              // Acquire (cost * 3) and then return zero
-                              .expectCapacity(TEST_BUCKET_CAPACITY - (TEST_EXCEPTION_COST * 3))
+                              // Acquire (cost * 2)
+                              .expectCapacity(TEST_BUCKET_CAPACITY - (TEST_EXCEPTION_COST * 2))
                               .expectState(DefaultRetryToken.TokenState.MAX_RETRIES_REACHED)
                               .expectThrows()
             , testCaseSupplier.apply("Fails when 4 exceptions are thrown out max of 3")
                               .configure(b -> b.maxAttempts(3))
                               .configure(b -> b.retryOnException(IllegalArgumentException.class))
                               .givenExceptions(IAE, IAE, IAE, IAE)
-                              // Acquire (cost * 3) and then return zero
-                              .expectCapacity(TEST_BUCKET_CAPACITY - (TEST_EXCEPTION_COST * 3))
+                              // Acquire (cost * 2)
+                              .expectCapacity(TEST_BUCKET_CAPACITY - (TEST_EXCEPTION_COST * 2))
```

Comment:
Note to reviewers: This changed since we now acquire capacity as the last step rather than at the beginning.